polynote-kernel/src/main/scala/polynote/data/Rope.scala

```diff
@@ -41,7 +41,13 @@ sealed abstract class Rope {
 
   def withEdit(edit: ContentEdit): Rope = edit.applyTo(this)
 
-  def withEdits(edits: List[ContentEdit]): Rope = edits.foldLeft(this)(_ withEdit _)
+  /*
```

Comment:
I should remove this comment... edits together in `ContentEdits` are actually considered to be dependent on all previous edits (still have to investigate whether that's reflected in monaco and the TBD rich text editor though)